### PR TITLE
hybrid mobile agent releases for ios agent update

### PR DESCRIPTION
**Pull request by ndesai-newrelic**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/release-notes/mobile-release-notes/capacitor-release-notes/capacitor-agent-137.mdx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+subject: Capacitor agent
+releaseDate: '2024-03-22'
+version: 1.3.7
+downloadLink: 'https://www.npmjs.com/package/@newrelic/newrelic-capacitor-plugin/v/1.3.7'
+---
+
+## New in this release
+
+- Updated native iOS Agent: We've upgraded the native iOS agent to version 7.4.10, which includes performance improvements and bug fixes.
```
Comment (ally-sassman) [inline src/content/docs/release-notes/mobile-release-notes/capacitor-release-notes/capacitor-agent-137.mdx:10]:
```suggestion
- Updated native iOS agent: We've upgraded the native iOS agent to version 7.4.10, which includes performance improvements and bug fixes.
```

File: src/content/docs/release-notes/mobile-release-notes/flutter-release-notes/flutter-agent-108.mdx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+subject: Flutter agent
+releaseDate: '2023-12-20'
```
Comment (ally-sassman) [inline src/content/docs/release-notes/mobile-release-notes/flutter-release-notes/flutter-agent-108.mdx:3]: @ndesai-newrelic Is this releaseDate correct?

**Review by ghost — CHANGES_REQUESTED** [review]

File: src/content/docs/mobile-monitoring/new-relic-mobile-unreal-engine/monitor-your-unreal-engine-application.mdx [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+<img
+  title="Guided install for monitoring Unreal Engine mobile apps"
+  alt="Screenshot of the guided install UI for Unreal Engine"
+  src={unrealGuidedInstall}
+  style={{ maxWidth: '70%' }}
+/>
+
+## Manual installation [#manual-install]
+
+If you need to install the plugin manually, follow these steps:
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-unreal-engine/monitor-your-unreal-engine-application.mdx:51]:
```suggestion
If you need to install the plugin manually, follow the steps below.
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-unreal-engine/monitor-your-unreal-engine-application.mdx [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+Add NewRelic to the list of dependencies in your game module’s .Build.cs file in order to allow interfacing with NewRelic from C++ code:
+
+**PrivateDependencyModuleNames.Add("NewRelic");**
+</Step>
+<Step>
+### Add application token(s) [#app-token]
+
+To enable New Relic to authenticate your Unreal Engine app's data, follow these steps:
+
+Configure your API key through Unreal Editor’s Project Settings window, under Plugins → NewRelic.
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-unreal-engine/monitor-your-unreal-engine-application.mdx:89]:
```suggestion
* In your project settings under **Plugins → NewRelic**, paste your app tokens(s) in the **New Relic App Token field(s)**. If you want to have separate New Relic mobile apps (one of Android and one for iOS), then you'll need to add both app tokens.
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-unreal-engine/monitor-your-unreal-engine-application.mdx [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+
+<img
+  title="Unreal Engine AppToken Setup"
+  alt="Add App Token in Unreal Engine Apps"
+  src={unrealAgentAppTokenSetup}
+  style={{ maxWidth: '70%' }}
+/>
+
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-unreal-engine/monitor-your-unreal-engine-application.mdx:97]:
```suggestion
    <img
      title="Unreal Engine AppToken Setup"
      alt="Add App Token in Unreal Engine Apps"
      src={unrealAgentAppTokenSetup}
      style={{ maxWidth: '70%' }}
    />

```

File: src/content/docs/mobile-monitoring/new-relic-mobile-unreal-engine/monitor-your-unreal-engine-application.mdx [hunk 6]
```diff
@@ -0,0 +1,233 @@
...
+Or in Config/DefaultEngine.ini:
+[/Script/NewRelic.NewRelicSDKSettings]
+newrelicAppToken=application-token
+newrelicAppTokenIOS=ios-application-token
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-unreal-engine/monitor-your-unreal-engine-application.mdx:101]:
```suggestion
In your `Config/DefaultEngine.ini` file:

```
[/Script/NewRelic.NewRelicSDKSettings]
newrelicAppToken=YOUR_APP_TOKEN
newrelicAppTokenIOS=YOUR_IOS_APP_TOKEN
```

```

@ndesai-newrelic Shouldn't newrelicAppToken=application-token be newrelicAndroidAppToken=application-token ?

File: src/content/docs/mobile-monitoring/new-relic-mobile-unreal-engine/monitor-your-unreal-engine-application.mdx [hunk 7]
```diff
@@ -0,0 +1,233 @@
...
+  </thead>
+
+  <tbody>
+    <tr>
+      <td id="crash-analysis">
+        Record breadcrumbs to track app activity that may be helpful for troubleshooting crashes.
+      </td>
+
+      <td>
+        [Record breadcrumbs](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-breadcrumb)
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-unreal-engine/monitor-your-unreal-engine-application.mdx:131]:
```suggestion
        [Record breadcrumbs](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-breadcrumb/#unreal)
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-unreal-engine/monitor-your-unreal-engine-application.mdx [hunk 8]
```diff
@@ -0,0 +1,233 @@
...
+
+    <tr>
+      <td id="creating">
+        Track a method as an interaction.
+      </td>
+
+      <td>
+        [Start interactions](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/start-interaction)
+
+        [Stop interactions](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/stop-interaction)
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-unreal-engine/monitor-your-unreal-engine-application.mdx:143]:
```suggestion
        [Start interactions](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/start-interaction/#unreal)

        [Stop interactions](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/stop-interaction/#unreal)
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-unreal-engine/monitor-your-unreal-engine-application.mdx [hunk 9]
```diff
@@ -0,0 +1,233 @@
...
+      </td>
+    </tr>
+
+    <tr>
+      <td id="create-custom">
+        Record custom metrics.
+      </td>
+
+      <td>
+        [Record custom metrics](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-custom-metrics/)
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-unreal-engine/monitor-your-unreal-engine-application.mdx:153]:
```suggestion
        [Record custom metrics](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-custom-metrics//#unreal)
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-unreal-engine/monitor-your-unreal-engine-application.mdx [hunk 10]
```diff
@@ -0,0 +1,233 @@
...
+      </td>
+    </tr>
+
+    <tr>
+      <td id="create-custom">
+        Record an exception.
+      </td>
+
+      <td>
+        [Record errors](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-handled-exceptions)
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-unreal-engine/monitor-your-unreal-engine-application.mdx:163]:
```suggestion
        [Record errors](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-handled-exceptions/#unreal)
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-unreal-engine/monitor-your-unreal-engine-application.mdx [hunk 11]
```diff
@@ -0,0 +1,233 @@
...
+
+          * [Record custom attributes](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/create-attribute)
+          * [Increment session attribute count](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/increment-session-attribute-count)
+          * [Remove an attribute](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/remove-attribute)
+          * [Remove all attributes](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/remove-all-attributes)
+          * [Record custom events](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-custom-events)
+          * [Set the maximum size of an event pool](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/set-max-event-pool-size)
+          * [Set maximum time the agent stores events in memory](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/set-max-event-buffer-time)
+          * [Get a current session's ID](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/current-session-id)
+          * [Set a custom user ID to associate with events and attributes](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/set-custom-user-id)
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-unreal-engine/monitor-your-unreal-engine-application.mdx:183]:
```suggestion
          * [Record custom attributes](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/create-attribute/#unreal)
          * [Increment session attribute count](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/increment-session-attribute-count/#unreal)
          * [Remove an attribute](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/remove-attribute/#unreal)
          * [Remove all attributes](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/remove-all-attributes/#unreal)
          * [Record custom events](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-custom-events/#unreal)
          * [Set the maximum size of an event pool](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/set-max-event-pool-size/#unreal)
          * [Set maximum time the agent stores events in memory](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/set-max-event-buffer-time/#unreal)
          * [Get a current session's ID](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/current-session-id/#unreal)
          * [Set a custom user ID to associate with events and attributes](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/set-custom-user-id/#unreal)
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-unreal-engine/monitor-your-unreal-engine-application.mdx [hunk 12]
```diff
@@ -0,0 +1,233 @@
...
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Shut down the agent.
+      </td>
+
+      <td>
+        [Shut down the agent](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/shut-down-agent)
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-unreal-engine/monitor-your-unreal-engine-application.mdx:195]:
```suggestion
        [Shut down the agent](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/shut-down-agent/#unreal)
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-unreal-engine/monitor-your-unreal-engine-application.mdx [hunk 13]
```diff
@@ -0,0 +1,233 @@
...
+      </td>
+    </tr>
+    <tr>
+      <td>
+        Run a test crash report.
+      </td>
+
+      <td>
+        [Test crash reporting](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/test-crash-reporting)
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-unreal-engine/monitor-your-unreal-engine-application.mdx:204]:
```suggestion
        [Test crash reporting](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/test-crash-reporting/#unreal)
```

**Review by ghost — CHANGES_REQUESTED** [review]

File: src/content/docs/mobile-monitoring/new-relic-mobile-unreal-engine/monitor-your-unreal-engine-application.mdx [hunk 14]
```diff
@@ -0,0 +1,231 @@
...
+* In your project settings under **Plugins → NewRelic**, paste your app tokens(s) in the **New Relic App Token field(s)**. If you want to have separate New Relic mobile apps (one of Android and one for iOS), then you'll need to add both app tokens.
+
+    <img
+    title="App token fields in Unreal editor"
+    alt="Screenshot of app token fields in project settings"
+    src={unrealAgentAppTokenSetup}
+    style={{ maxWidth: '70%' }}
+    />
+
+In your `Config/DefaultEngine.ini` file:
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-unreal-engine/monitor-your-unreal-engine-application.mdx:98]: @ndesai-newrelic What's supposed to follow this?

**Comment by ally-sassman** [comment]
netlify build

File: src/content/docs/mobile-monitoring/new-relic-mobile-unreal-engine/monitor-your-unreal-engine-application.mdx [hunk 15]
```diff
@@ -0,0 +1,231 @@
...
+/>
+
+<figcaption>
+  <DoNotTranslate>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile > (select an app) > Summary**</DoNotTranslate>: View Unreal Engine data, and errors, and monitor how your app is performing over time.
+</figcaption>
+
+
+## (Recommended) Guided installation [#guided-install]
+
+To install the Unreal Engine Plugin, follow our [guided install](https://onenr.io/0gR70k5oZjo), located directly in the UI.
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-unreal-engine/monitor-your-unreal-engine-application.mdx:40]: @ndesai-newrelic This takes me to the Flutter setup, and when I tried to search for Unreal, nothing came up.

**Review by ghost — APPROVED** [review]

**Pull request merged**